doctest/doctest.h

```diff
@@ -445,7 +455,6 @@ DOCTEST_GCC_SUPPRESS_WARNING_POP
 #ifndef DOCTEST_CONFIG_INCLUDE_TYPE_TRAITS
 #define DOCTEST_CONFIG_INCLUDE_TYPE_TRAITS
 #endif // DOCTEST_CONFIG_INCLUDE_TYPE_TRAITS
-#include <cmath>
 #include <cstddef>
 #include <ostream>
 #include <istream>
```

Comment:
`#include<iosfwd>` might be sufficient here. Because it declares but not defines both ostream and istream and its inclusion times are non-existing comparing to ostream and istream see https://artificial-mind.net/projects/compile-health/
But I'm not sure if DOCTEST_CONFIG_USE_STD_HEADERS is properly tested.